assets/adept.yaml

```diff
@@ -71,3 +71,27 @@
     is out-of-scope for the abilities of this model.
   monitoring: ''
   feedback: https://huggingface.co/adept/fuyu-8b/discussions
+- type: model
+  name: Fuyu Heavy
+  organization: Adept
+  description: Fuyu Heavy is a new multimodal model designed specifically for digital agents.
+  created_date: 2024-01-24
+  url: https://www.adept.ai/blog/adept-fuyu-heavy
+  model_card: none
+  modality: image, text; text
+  analysis: Evaluated on the MMLU, GSM8K, MATH, and HumanEval benchmarks. According to these benchmarks, Fuyu-Heavy is, as of release, the strongest multimodal model trained outside of Google or OpenAI.
+  size: 
+    explanation: The size of the model is 10-20 times smaller than GPT-4V and Gemini Ultra, as per announcement.
+    value: unknown
+  dependencies: []
+  training_emissions: unknown
+  training_time: unknown
+  training_hardware: unknown
+  quality_control: none
+  access: closed
+  license: CC BY NC 4.0
```

Comment:
where does it say this/i don't think this is right (CC BY is a data license, so it would be weird a priori to be a model license)